pkg/minishift/openshift/service.go

```diff
@@ -72,7 +89,7 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 }
 
 // Get the available routes to user
-func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
+func GetServiceURLs(serviceListNamespace string, ip string) ([]ServiceURL, error) {
```

Comment:
Do we really have to pass the IP? This could be added at the table displaying level. We know that NodePorts are exposed in our context on the VM IP.